recipes/configs/llama3/70B_full.yaml

```diff
@@ -14,6 +14,12 @@
 #   tune run --nproc_per_node 8 full_finetune_distributed --config llama3/70B_full checkpointer.checkpoint_dir=<YOUR_CHECKPOINT_DIR>
 #
 # This config is only tested on an 8xA100 machine.
+#
+# !!!!!!!!!!!!!
+# !!!!!!!!!!!!!
```

Comment:
nit: I would take the !!!! out personally

Reply:
i am afraid that users would ignore it without the "!", and not understand why it crashes